lib/yamatanooroti/vterm.rb

```diff
@@ -89,7 +92,10 @@ def close
   private def sync(wait = @wait)
     sync_until = Time.now + @timeout
     while @pty_output.wait_readable(wait)
-      vterm_write(@pty_output.read_nonblock(65536))
+      @pty_output.wait_readable(sync_until - Time.now) if @maybe_read
```

Comment:
`@pty_output.wait_readable(sync_until - Time.now)` seems to have no effect.
This code is inside `while @pty_output.wait_readable(wait)` so `@pty_output` is already readable.